src/tiledb/cloud/soma/ingest.py

```diff
@@ -208,8 +283,11 @@ def ingest_h5ad(
     import tiledbsoma.logging as somalog
     from tiledbsoma import io
 
+    # XXX parameterize
```

Comment:
did you intend to follow up on this?

Reply:
As above -- this is a WIP on which I very much need help